src/ImageSharp/PixelFormats/PixelBlenders/PorterDuffFunctions.cs

```diff
@@ -0,0 +1,242 @@
+﻿// <copyright file="PorterDuffFunctions.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.PixelFormats.PixelBlenders
+{
+    using System.Numerics;
+    using System.Runtime.CompilerServices;
+
+    /// <summary>
+    /// Collection of Porter Duff alpha blending functions
+    /// </summary>
+    /// <remarks>
+    /// These functions are designed to be a general solution for all color cases,
+    /// that is, they take in account the alpha value of both the backdrop
+    /// and source, and there's no need to alpha-premultiply neither the backdrop
+    /// nor the source.
+    /// Note there are faster functions for when the backdrop color is known
```

Comment:
Has anyone benchmarked this? The existing functions that implement  `Vector4BlendTransforms.cs` are per-pixel. I think we should actually scrap that class + `Rgba32` and `RgbaVector` transforms and us a single set of functions throughout.